src/FsAutoComplete.Core/Commands.fs

```diff
@@ -764,7 +765,7 @@ module Commands =
         return Choice1Of2(declarationRanges, usageRanges)
 
       | SymbolDeclarationLocation.Projects (projects, isInternalToProject) ->
-        let symbolUseRanges = ImmutableArray.CreateBuilder()
+        let symbolUseRanges = ConcurrentBag<_>()
```

Comment:
ImmutableArray.Builder isn't thread safe (ImmutableArray _is_ but the Builder _isn't_). Rather than rewriting the whole section, decided to use ConcurrentBag.